### ISSUE TITLE
Tesseract 4.1.0-rc2 incorrectly reports version

### ISSUE THREAD

**Issue by mikegerber** [tesseract-ocr/tesseract#2470]
### Environment

* **Tesseract Version**: 4.1.0-rc2
* **Commit Number**: built from tar
* **Platform**: Linux localhost.localdomain 5.0.17-200.fc29.x86_64 #1 SMP Mon May 20 15:39:10 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

### Current Behavior:
Reports 4.1.0-rc1:
~~~
$ tesseract --version | head -1
tesseract 4.1.0-rc1
~~~

### Expected Behavior:
It should report 4.1.0-rc2.

### Suggested Fix:
I report this so this issue is known if someone reports another issue, i.e. that 'rc1' could actually mean 'rc2'. I expect the next release to have a correct version :)

**Comment by zdenop** [comment]
Please provide all details how did you installed tesseract

**Comment by stweil** [comment]
See https://github.com/tesseract-ocr/tesseract/blob/4.1.0-rc2/VERSION. The `VERSION` file in the tagged 4.1.0-rc2 release includes the old version. That cannot be fixed in the distribution, just reported. @mikegerber, thank you for reporting this. You can fix the VERSION file locally before starting the build process.

**Comment by mikegerber** [comment]
Same for rc3 btw:

~~~
% cat tesseract-4.1.0-rc3/VERSION 
4.1.0-rc2
~~~
